components/collective-page/index.js

```diff
@@ -323,7 +323,11 @@ class CollectivePage extends Component {
         return (
           <SectionOurTeam
             collective={this.props.collective}
-            coreContributors={this.props.coreContributors}
+            coreContributors={
+              this.props.collective.type === 'EVENT'
+                ? this.props.collective.parentCollective.coreContributors
+                : this.props.coreContributors
+            }
```

Comment:
Projects have parent too. In the "Edit members" page, we defined that if there's a parent then the members should be defined there: https://github.com/opencollective/opencollective-frontend/blob/master/components/edit-collective/sections/Members.js#L372. It's also how it is enforced in the permissions. 

So we could directly write:
```suggestion
            coreContributors={
              this.props.collective.parentCollective?.coreContributors || this.props.coreContributors
            }
```